### PR TITLE
Surface overridden storage settings on the Instance Type tab.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
screen shot 2015-12-18 at 1 27 08 pm

File: app/scripts/modules/core/serverGroup/configure/common/instanceTypeDirective.html [hunk 1]
```diff
@@ -34,13 +34,17 @@
           ng-click="instanceTypeCtrl.selectInstanceType(instanceType)">
         <td>
           <help-field ng-if="instanceType.unavailable" key="instanceType.unavailable"></help-field>
+          <help-field ng-if="instanceType.helpFieldKey" key="{{instanceType.helpFieldKey}}"></help-field>
```
Comment (duftler) [inline app/scripts/modules/core/serverGroup/configure/common/instanceTypeDirective.html]: Also added this back.

**Comment by anotherchrisberry** [comment]
LGTM - needs a rebase

**Comment by duftler** [comment]
Rebased.

**Pull request merged**